lib/platform/common.ts

```diff
@@ -148,6 +153,7 @@ export interface EnsureCommentConfig {
   number: number;
   topic: string;
   content: string;
+  force?: boolean;
```

Comment:
The `force` option allows us to add a comment regardless of whether it exists already, which is necessary because the automerge pr-comment might not be authored by renovate and therefore cannot be removed by renovate.